system_baseline/views/v1.py

```diff
@@ -113,16 +146,23 @@ def _delete_baselines(baseline_ids):
         SystemBaseline.account == account_number, SystemBaseline.id.in_(baseline_ids)
     )
 
+    message = "reading baselines"
+    current_app.logger.audit(message, request=request)
     full_results = query.all()
     if len(full_results) < len(baseline_ids):
         fetched_ids = {str(result.id) for result in full_results}
         missing_ids = set(baseline_ids) - fetched_ids
+
+        message = "ids [%s] not available to delete" % ", ".join(missing_ids)
+        current_app.logger.audit(message, request=request, success=False)
         raise HTTPError(
-            HTTPStatus.NOT_FOUND,
-            message="ids [%s] not available to delete" % ", ".join(missing_ids),
+            HTTPStatus.NOT_FOUND, message=message,
         )
 
     query.delete(synchronize_session="fetch")
+
+    message = "deleting baseline"
+    current_app.logger.audit(message, request=request)
     db.session.commit()
 
```

Comment:
Do you need to add a "deleted baselines" paired message for your above "deleting baselines" message as you did with the other deletes?

Reply:
Yes, thank you... I need only the later one, when it succeeds, see my other comment.